public-ui/src/app/data-browser/views/quick-search/quick-search.component.ts

```diff
@@ -232,7 +232,7 @@ export class QuickSearchComponent implements OnInit, OnDestroy {
     this.surveyResults = results.surveyModules;
     // TODO remove this filter when the feature flag is turned off or to debug
     if (!this.cope) {
-      this.surveyResults = results.surveyModules.filter(sm => sm.conceptId !== 1333342);
+      // this.surveyResults = results.surveyModules.filter(sm => sm.conceptId !== 1333342);
```

Comment:
do we want to keep this until we are done and ready for QA so it doesn't show up on the test??

Reply:
Thanks. Missed it as part of debugging.